CMakeLists.txt

```
@@ -29,6 +29,8 @@ option(GINKGO_WITH_IWYU "Make Ginkgo call `iwyu` (Include What You Use) to find
 option(GINKGO_CHECK_CIRCULAR_DEPS
     "Enable compile-time checks detecting circular dependencies between libraries and non-self-sufficient headers."
     OFF)
+set(GINKGO_CONFIG_LOG_VERBOSITY "0" CACHE STRING
```

suggestion: maybe change the variable type
```suggestion
set(GINKGO_CONFIG_LOG_VERBOSITY "0" CACHE BOOL
```

---

if it only need on/off option, maybe use command `option` and the name `GINKGO_CONFIG_LOG_DETAIL`

---

I made it a variable, in case we wanted to write more levels into it, but I guess that is not strictly necessary. I have made it an `option` now.